roles/pulp/defaults/main.yaml

```
@@ -0,0 +1,10 @@
+---
+pulp_image: pulp/pulp:stable
```

```suggestion
pulp_image: quay.io/pulp/pulp:stable
```

podman wants fully qualified container names